HLTriggerOffline/Scouting/plugins/ElectronEfficiencyPlotter.h

```diff
@@ -0,0 +1,60 @@
+#ifndef ElectronEfficiencyPlotter_H
+#define ElectronEfficiencyPlotter_H
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "DQMServices/Core/interface/DQMEDHarvester.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include <FWCore/Framework/interface/ESHandle.h>
+#include <FWCore/Framework/interface/Event.h>
+#include <FWCore/Framework/interface/MakerMacros.h>
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include <FWCore/Framework/interface/LuminosityBlock.h>
+
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/Framework/interface/Run.h"
+
+#include <memory>
+#include <iostream>
+#include <string>
+#include <vector>
+#include <map>
+#include "TH1F.h"
+
+class ElectronEfficiencyPlotter : public DQMEDHarvester {
+public:
+  // Constructor
+  ElectronEfficiencyPlotter(const edm::ParameterSet& ps);
+  // Destructor
+  ~ElectronEfficiencyPlotter() override;
```

Comment:
```suggestion
  ~ElectronEfficiencyPlotter() override = default;
```

(and remove from the implementation)